pyteal/ir/tealsimpleblock.py

```diff
@@ -7,10 +7,14 @@
 class TealSimpleBlock(TealBlock):
     """Represents a basic block of TealComponents in a graph that does not contain a branch condition."""
 
-    def __init__(self, ops: List[TealOp]) -> None:
+    def __init__(self, ops: List[TealOp], traceback: list[str] = []) -> None:
```

Comment:
nit: do not use `[]` as default argument, they are **mutable default argument**. Please consider using `None`